blog/20240426-simplex-legally-binding-transparency-v5-7-better-user-experience.md

```diff
@@ -10,4 +9,95 @@ permalink: "/blog/20240426-simplex-legally-binding-transparency-v5-7-better-user
+
+## What's new in v5.7
+
+This release focus is improving the app usability, and preparing the foundation for v5.8 that will provide an in-built protection of user IP addresses when connecting to unknown file and messaging servers, reducing the need to use Tor (which would still remain supported via SOCKS proxy, for additional privacy).
+
+### Quantum resistant encryption
+
+TODO picture
+
+We [wrote before](./20240314-simplex-chat-v5-6-quantum-resistance-signal-double-ratchet-algorithm.md) about how quantum resistant encryption was added to SimpleX Chat and also about other properties of end-to-end encryption, possible attacks on its security and known mitigations.
+
+Quantum resistant encryption will now be enabled by default in all direct chats. For the new conversations it will be enabled from the beginning, and for the existing conversations it will be agreed after you exchange several messages with your contacts - you will see a notice in the conversation when it happens.
+
+You can still safely downgrade the app to an earlier version if needed, as v5.6 already supports quantum resistant encryption.
+
+With the users who have an earlier version, the app will work using the conventional encryption, which is still very secure. It's important that we augmented the conventional encryption with post-quantum cryptographic algorithm rather than replaced it, using a hybrid construction as recommended by the cryptography experts.
+
+The groups currently do not support quantum resistant encryption yet - we plan to add it in the future to small groups.
+
+### Forward and save messages
+
+TODO picture
+
+You can now save received messages to private notes and forward them to your contacts and groups. This is both more convenient than copy-pasting the messages, and also more private - you can forward files and preserve a disappearing file in your private notes without saving them outside of the app.
+
+You can see and navigate to the original source of the message via the message information, but the recipient of the message can only see that it was forwarded, but not from which conversation - in this way you can show that the message was quoted from another source without revealing the source ([Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_rule)).
+
+### In-call sounds
+
+This was the most frequent request of the users who use SimpleX Chat for audio and video calls - to add sound indication to the connection progress, as happens in all other apps. This release added these sounds and also vibration when call connects and disconnects.
+
+### Customize shapes of profile images in the app
+
+To make the interface more customizable, you can now change the shapes of profile images to anything from squares with sharp corners to circles, as before, via the Appearance menu in the app Settings. The default shape of profile images is changed to squares with rounded corners.
```

Comment:
this doesn't make sense structurally, it would be ok in such ways:
- "To make the interface more customizable, we added this feature"
- or "To make the interface more customized/custom (?), you can now change the shapes"
users can't make the interface customizable (unless there was another option enabling/disabling customization settings to make the interface customizable)